teps/0029-step-workspaces.md

```diff
@@ -57,6 +69,99 @@ that has access to those contents as well.
 - A Task author can still use the volume "hack" to attach `Workspaces` to `Sidecars` in
 combination with the feature proposed here.
 
+## Proposal
+
+Add `workspaces` lists to Steps and Sidecars. Here's the existing YAML and behaviour:
+
+```yaml
+spec:
+  workspaces:
+  - name: foo
+  steps:
+  - image: ubuntu
+    script: # ... This Step automatically mounts "foo" workspace.
+  - image: ubuntu
+    script: # ... And so does this Step.
+```
+
+And here's a Task using the new Step Workspaces feature:
+
+```yaml
+spec:
+  workspaces:
+  - name: foo
```

Comment:
Is the `foo` workspace required to appear in here to be also declared in the `step` or for example sake, should it be named differently ?

Reply:
(got my answer before, it is required)